### DISCUSSION TITLE
How to convert to Chinese

### DISCUSSION THREAD

**Discussion by zuohuaijun** [elsa-workflows/elsa-core#4100] [Q&A]

**Comment by luowenhao-github** [comment]
你想把什么转换为中文

**Comment by A-k-ing** [comment]
怎么把流程定义，流程界面设置成中文的

**Comment by A-k-ing** [comment]
没有找到参数可以设置项目的本地化语言，或者如何将界面的英文按钮，标题，在一个对照表中设置

**Comment by XiangZou2** [comment]
有方案了吗，哈哈

**Comment by sfmskywalker** [comment]
Good point, we still need to implement (at least) UI localisation.

**Comment by sfmskywalker** [comment]
Hi everyone,

[Elsa Studio](https://github.com/elsa-workflows/elsa-studio) now supports localization, and we're looking for contributors to help create translations. We use [Weblate](https://hosted.weblate.org/projects/elsa-studio/) to make managing translations simple and collaborative. If you're interested in contributing Chinese translations, please sign up for a Weblate account and share your username or email address with me.

We'd greatly appreciate your help!

**Reply by pangfd** [reply]
How to switch the language of  Elsa-Studio ?

**Reply by sfmskywalker** [reply]
https://deepwiki.com/search/how-to-switch-the-language-of_47704ab4-d1ed-42d0-be4a-dddf726671f1?mode=fast

**Reply by pangfd** [reply]
Thx !

**Comment by sfmskywalker** [comment]
Closing as answered. Elsa Studio's localization module is documented in the repository, and the follow-up confirmed the guidance was helpful. If language switching still fails in a current version, please open a new report with the Studio host type, package version, and localization configuration.
